cmd/workload/filtertestgen.go

```diff
@@ -71,6 +71,9 @@ func filterGenCmd(ctx *cli.Context) error {
 		f.updateFinalizedBlock()
 		query := f.newQuery()
 		query.run(f.client, nil)
+		if query.Err == errPrunedHistory {
+			continue
```

Comment:
I think we should error here. It's not really possible to generate meaningful tests on a pruned node.

Reply:
I changed it so that the generator exits on errors instead of collecting them into a separate file. It also fails on history pruning errors. Now I think it doesn't make sense to collect errors into a file during generation as the test vectors should be generated in a correct environment. On the other hand it does make sense to collect errors during the performance test so I added it there (the `--errors` flag was there in `filterperf` anyways but it didn't do anything until now).